packages/contract-addresses/CHANGELOG.json

```diff
@@ -1,4 +1,13 @@
 [
+    {
+        "version": "6.18.0",
+        "changes": [
+            {
+                "note": "Goerli and Mumbai updated verified contracts"
+                "pr": 
```

Comment:
PR number is missing.

Reply:
fixed and bumped above the previously published version